### DISCUSSION TITLE
Tensorflow models minimum duration

### DISCUSSION THREAD

**Discussion by PRamoneda** [MTG/essentia#1128] [Essentia Models]
hi @palonso!

Which is the minimum sound duration that tensorflow models need?

How can I tune this?

**Comment by palonso** [comment]
@PRamoneda,
The minimum duration depends on the model's receptive field (VGGish: 1s, EffnetDiscgos: 2s, MusiCNN: any).
If the audio is too short, I recommend applying padding before processing.
